arcgis-ios-sdk-samples/Maps/Display device location/DisplayLocationSettingsViewController.swift

```diff
@@ -0,0 +1,128 @@
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class DisplayLocationSettingsViewController: UITableViewController {
```

Comment:
This settings view is so simple, it would have been nice to keep everything in one view. You could have used a picker for the auto pan mode and done away with the navigation controller.

I suppose it's not worth changing since you've already implemented it, but something to think about in the future.

Reply:
I'm trying to be consistent with the controls.